src/pde_solve.jl

```diff
@@ -59,5 +66,87 @@ function DiffEqBase.solve(
 
     Flux.train!(loss, ps, data, opt; cb = cb)
 
-    save_everystep ? iters : u0(X0)[1]
+
+    if give_limit == false
+        save_everystep ? iters : u0(X0)[1]
+    else
+        A = prob.kwargs.data.A
+        u_domain = prob.kwargs.data.u_domain
+
+        ## UPPER LIMIT
+        m1 = trajectories_upper
+        sdeProb = SDEProblem(μ , σ , X0 , prob.tspan)
+        ensembleprob = EnsembleProblem(sdeProb)
+        sim = solve(ensembleprob, sdealg, ensemblealg, dt=dt,trajectories=m1,adaptive=false)
+        function sol_high()
+            Uo = []
```

Comment:
This should be strictly typed or else I can see this causing some performance issues. `eltype(sim.u)[]`

Reply:
doing the following as a map and then `reduce(vcat,U)` could be an easy way to generically handle it.